recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -96,9 +96,13 @@ def layout(self):
 
     def requirements(self):
         self.requires("robin-hood-hashing/3.11.5")
-        # TODO: set private=True, once the issue is resolved https://github.com/conan-io/conan/issues/9390
-        self.requires(self._require("spirv-tools"), private=not hasattr(self, "settings_build"))
-        self.requires(self._require("vulkan-headers"))
+        self.requires(self._require("spirv-headers"))
+        if Version(conan_version).major < "2":
+            # TODO: set private=True, once the issue is resolved https://github.com/conan-io/conan/issues/9390
```

Comment:
🤔 there very little reason things should be exposed like this... its an old PR but could you elaborate? 🙏

Reply:
I don't remember, and I'm not even sure do have written this specific block.